nltk/corpus/reader/verbnet.py

```diff
@@ -70,30 +71,32 @@ def __init__(self, root, fileids, wrap_etree=False):
     """Regular expression used by ``_index()`` to quickly scan the corpus
        for basic information."""
 
-    def lemmas(self, classid=None):
+    def lemmas(self, vnclass=None):
```

Comment:
You have tests, but it would be nice to have examples within the methods (lemmas, wordnetids).